www-plugins/passff/passff-1.1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="zx2c4 pass manager extension for Firefox (Host Binary)"
+HOMEPAGE="https://github.com/passff/passff"
+SRC_URI="
+		https://github.com/passff/passff/releases/download/${PV}/passff.json -> ${P}.json
+		https://github.com/passff/passff/releases/download/${PV}/passff.py -> ${P}.py
+"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=dev-lang/python-3.4.0:*"
+
+TARGET_DIR="${EROOT}usr/lib/mozilla/native-messaging-hosts"
+
+S="${DISTDIR}"
+
+src_compile() {
+	sed -i "s|PLACEHOLDER|${TARGET_DIR}/${PN}.py|" "${P}.json"
+}
+
+src_install() {
+	insinto "${TARGET_DIR}"
+	newins "${P}.json" passff.json
+	newins "${P}.py" passff.py
+	chmod +x "${D}${TARGET_DIR}/passff.py"
```

Comment:
`EROOT` is not valid here. Also `fperms` is provided as replacement for `chmod` that works inside `ED`.